src/main/java/org/jenkinsci/test/acceptance/docker/DockerFixture.java

```diff
@@ -29,14 +29,24 @@
     String id();
 
     /**
-     * Ports that are exposed from this fixture.
+     * TCP ports that are exposed from this fixture.
      *
      * <p>
      * When a container is started, these ports from the container are mapped to
      * random ephemeral ports on the host. The actual ephemeral port number can
      * be retried at runtime via {@link DockerContainer#port(int)}.
      */
     int[] ports() default {};
+
+    /**
+     * UDP ports that are exposed from this fixture.
+     *
+     * <p>
+     * When a container is started, these ports from the container are mapped to
+     * random ephemeral ports on the host. The actual ephemeral port number can
+     * be retried at runtime via {@link DockerContainer#port(int)}.
```

Comment:
`docker-inspect` offers no separate mapping for UDP ports?

Reply:
Correct, I did not know it. See the output:

```
pegaso:53310 fbelzunc$ docker port fc46392cb17f 53
0.0.0.0:53
pegaso:53310 fbelzunc$ docker port fc46392cb17f 53/udp
0.0.0.0:53
pegaso:53310 fbelzunc$ docker port fc46392cb17f
138/tcp -> 0.0.0.0:138
3268/tcp -> 0.0.0.0:3268
3269/tcp -> 0.0.0.0:3269
53/tcp -> 0.0.0.0:53
88/tcp -> 0.0.0.0:88
53/udp -> 0.0.0.0:53
135/tcp -> 0.0.0.0:135
139/tcp -> 0.0.0.0:139
389/tcp -> 0.0.0.0:389
445/tcp -> 0.0.0.0:445
464/tcp -> 0.0.0.0:464
```